src/library/components/atoms/as-notification/AsNotification.vue

```diff
@@ -4,7 +4,12 @@
     class="as-notification"
     :class="{ 'is-small': $props.isSmall }"
   >
-    <span class="as-notification__quantity">{{ props.quantity }}</span>
+    <span class="as-notification__quantity">
+      {{
+        // eslint-disable-next-line @intlify/vue-i18n/no-raw-text
+        props.quantity >= 100 ? '+99' : props.quantity
```

Comment:
Why is this logic on the template and no on the code?

Reply:
I have added this logic in the template because I think it is better to avoid using a ref() to listen for changes prop in this case. I take a look at trying to make a simple logic in the script. Thanks!!!

Reply:
Thanks for the suggestion, I have used a calculated value to do so.